### PR TITLE
Obscure scripting language used by mame/quine-relay

### PR DESCRIPTION
**Pull request by markjreed**

### CODE REVIEW

File: Library/Formula/spl.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Spl < Formula
+  homepage "http://www.clifford.at/spl/"
+  url "http://www.clifford.at/spl/releases/spl-1.0pre6.tar.gz"
```
Comment (DomT4) [inline Library/Formula/spl.rb]: What does the ` pre ` stand for? Pre-Release?

File: Library/Formula/spl.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "737658aea624f17ede8bb965b89cbd4acf40148906c55d3eebc8cc34d1755c30"
+  version "1.0pre6"
+
+  depends_on 'pcre' => :build
```
Comment (DomT4) [inline Library/Formula/spl.rb]: Double quotes everywhere.

File: Library/Formula/spl.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on 'postgres' => :build
+
+  fails_with :clang do
+    build 602
+    cause <<-EOS.undent
+        Requires nested functions, which are supported only by GCC.
```
Comment (DomT4) [inline Library/Formula/spl.rb]: Looks like this might have been intended by tab instead of spaces?

File: Library/Formula/spl.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    EOS
+  end
+
+  def install
+    inreplace "GNUmakefile", 'prefix = /usr/local', "prefix = #{prefix}"
```
Comment (DomT4) [inline Library/Formula/spl.rb]: Instead of ` inreplace`-ing can we just not pass ` prefix=#{prefix} ` to ` make install `? Does that not work here?

**Comment by asparagui** [comment]
>Warning: SPL is unmaintained. I do not recommend it for new projects.

File: Library/Formula/spl.rb (hunk 1 above)
Comment (markjreed) [inline Library/Formula/spl.rb]: Yup, added comment.

Comment (DomT4) [inline Library/Formula/spl.rb]: Sadly, even if development has ground to a sudden halt, pre-release may mean it ends up in ` devel-only `. I'll let one of the maintainers here make the decision though - I have no "power" over the core :wink:

**Comment by mikemcquaid** [comment]
Seems a better candidate for a tap than core if it's an unmaintained prerelease.

**Comment by jacknagel** [comment]
Agree, passing on this as-is.

**Pull request closed without merging**